<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormsPage</class>
 <widget class="QWidget" name="FormsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>860</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>296</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="PassPortGird">
    <item row="0" column="0">
     <widget class="ElevatedCardWidget" name="PassPortCard">
      <widget class="TitleLabel" name="PassPortTitle">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>5</y>
         <width>146</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>通行证申请</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="PassPortButton">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>400</y>
         <width>186</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>填写申请表</string>
       </property>
       <property name="icon">
        <iconset theme="FluentIcon.SHARE">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="hasIcon" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>55</y>
         <width>251</width>
         <height>326</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="PassPortQRCode_Grid">
        <item row="0" column="0">
         <widget class="PixmapLabel" name="QRcode_PassPort"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="ReportGird">
    <item row="0" column="0">
     <widget class="ElevatedCardWidget" name="ReportCard">
      <widget class="TitleLabel" name="Report">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>256</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>BUG/更新 意见与反馈</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="HyperlinkLabel" name="ReportLink">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>361</width>
         <height>26</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>【腾讯文档】月梦初晓MC 反馈&amp;bug报告</string>
       </property>
       <property name="url">
        <url>
         <string>https://docs.qq.com/form/page/DTWVTUmxpZkVveWJN</string>
        </url>
       </property>
       <property name="underlineVisible">
        <bool>true</bool>
       </property>
       <property name="underline" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="PrimaryToolButton" name="ReportLinkButton">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>55</y>
         <width>76</width>
         <height>36</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>155</y>
     <width>241</width>
     <height>316</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="MoreInfoGird">
    <item row="0" column="0">
     <widget class="ElevatedCardWidget" name="MoreInfoCard">
      <widget class="TitleLabel" name="MoreInfo">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>5</y>
         <width>146</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>更多信息</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="MoreInfoButton">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>265</y>
         <width>186</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>跳转外部文档</string>
       </property>
       <property name="icon">
        <iconset theme="FluentIcon.SHARE">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="hasIcon" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>55</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="MoreInfoQRCode_Gird">
        <item row="0" column="0">
         <widget class="PixmapLabel" name="MoreInfo_QRcode"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>155</y>
     <width>241</width>
     <height>316</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="AboutGird">
    <item row="0" column="0">
     <widget class="ElevatedCardWidget" name="AboutLauncherCard">
      <widget class="TitleLabel" name="AboutLanucher">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>5</y>
         <width>146</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>关于启动器</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="GithubButton">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>265</y>
         <width>186</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>跳转Github</string>
       </property>
       <property name="icon">
        <iconset theme="FluentIcon.SHARE">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="hasIcon" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="IconWidget" name="GitHub_Gird">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>60</y>
         <width>161</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryToolButton</class>
   <extends>ToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PixmapLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElevatedCardWidget</class>
   <extends>SimpleCardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HyperlinkLabel</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
